cpp/src/IceSSL/OpenSSLTransceiverI.cpp

```diff
@@ -171,7 +171,7 @@ OpenSSL::TransceiverI::initialize(IceInternal::Buffer& readBuffer, IceInternal::
             // Hostname verification was included in OpenSSL 1.0.2
             //
 #if defined(OPENSSL_VERSION_NUMBER) && OPENSSL_VERSION_NUMBER >= 0x10002000L
-            if(_engine->getCheckCertName() && !_host.empty() && (sslVerifyMode & SSL_VERIFY_PEER))
+            if(_engine->getCheckCertName() && !_host.empty())
```

Comment:
So like C#, we weren't doing the hostname check with IceSSL.VerifyPeer=0 when using OpenSSL >= 1.0.2, shouldn't we mention this like for C# in the ChangeLog?